### ISSUE TITLE
[Feature Request] Change/Update the Python version for the promptflow-runtime:latest docker image

### ISSUE THREAD

**Issue by WilliamWatsonEAI** [microsoft/promptflow#3817]
Would it be possible to select of update the Python version used in the Azure runtime? The current promptflow-runtime-stable release uses Python 3.9. This is causing lots of issues with the more recent openai package as well as promptflow-tracing. 

Workarounds have been posted over the past month to avoid the issue such as [3751](https://github.com/microsoft/promptflow/issues/3751) [3806](https://github.com/microsoft/promptflow/issues/3806). 

These work for resolving the issue locally and allows for deployment to an online managed endpoint via docker. But the latest which requires tracing for PF to be disabled is not viable longterm. 

I note that @luigiw mentioned that promptflow-tracing 1.16.1 resolves this issue, but this version seems to be incompatible with the latest promptflow-runtime.

Basically, I was wondering if there are plans to upgrade the promptflow-runtime image from 3.9 to 3.10/11?
